rbc/typesystem.py

```diff
@@ -1212,6 +1222,9 @@ def _demangle(s):
 if nb is not None:
     class Boolean1(nb.types.Boolean):
 
+        def __name__(self):
+            return 'Boolean1'
+
```

Comment:
Which part of the PR uses `__name__` method? Also, `__name__` is usually an attribute or property, not a method.
Remove if not needed. The same applies to `Boolean8` below.

Reply:
They're were required before when I was constructing a type on `omniscidb.py` but I guess it can be removed now

Reply:
Yes, let's remove it if not needed.